src/v40/EntryPoint40.js

```diff
@@ -27,26 +27,34 @@ const { ToucheggConfig } = SRC.touchegg.ToucheggConfig;
 const { AllowedGesture } = SRC.utils.AllowedGesture;
 const { logger } = SRC.utils.Logger;
 
+const ExtensionUtils = imports.misc.extensionUtils;
+const Me = ExtensionUtils.getCurrentExtension();
+const Convenience = Me.imports.convenience;
+
 /**
  * Entry point for GNOME Shell 40.
  */
 class EntryPoint40Class extends GObject.Object {
   static start() {
+    let settings = Convenience.getSettings('org.gnome.shell.extensions.x11gestures');
```

Comment:
`npm run lint` shows a few errors, prefer const if variables don't change:

```suggestion
    const settings = Convenience.getSettings('org.gnome.shell.extensions.x11gestures');
```

Reply:
Done. @JoseExposito do you prefer to resolve conversation yourself or should I?

Reply:
Cool, thank you. GitHub forces now to approve workflow runs on PRs, so I approved the lint job on this one so it's easier for you.